packages/adblocker/src/resources.ts

```diff
@@ -297,8 +297,27 @@ export default class Resources {
     }
   }
 
-  public getResource(name: string): { body: string; contentType: string; dataUrl: string } {
-    const { body, contentType } = this.resourcesByName.get(name) || getResourceForMime(name);
+  public getResource(name: string): {
+    filename: string;
+    body: string;
+    contentType: string;
+    dataUrl: string;
+  } {
+    let resource:
+      | {
```

Comment:
Nit. Is this a prettier thing to put a leading `|` there?

Reply:
Yes, it's.